core/dateUtils.js

```diff
@@ -108,10 +108,22 @@ export const isValidDateInFormat = (dateStr, format) => {
   return fnsIsValid(parsed)
 }
 
+const substractTimezoneOffset = (date) => {
+  const timezoneOffset = date.getTimezoneOffset() * 60000
+  return new Date(date.getTime() - timezoneOffset)
+}
+
+const addTimezoneOffset = (date) => {
+  const timezoneOffset = date.getTimezoneOffset() * 60000
+  return new Date(date.getTime() + timezoneOffset)
+}
+
 export const formatDateISO = (date) => (date ? format(date, formats.dateISO) : null)
 
 export const formatDateTimeDefault = (date) => (date ? format(date, formats.datetimeDefault) : null)
 
+export const formatDateTimeISO = (date) => (date ? format(addTimezoneOffset(date), formats.datetimeISO) : null)
```

Comment:
The validation of date null could be extracted to the format function

Reply:
right again!